portfolio/src/main/java/com/google/sps/data/DataComment.java

```diff
@@ -1,17 +1,16 @@
 
 package com.google.sps.data;
 
-import java.util.Date;
+import java.util.ArrayList;
+import java.util.List;
 
 /** Class containing server statistics. */
-public final class DataComment {
+public class DataComment {
```

Comment:
Why was the 'final' removed? In general prefer to mark classes as final so that you won't have to worry about dealing with potential subclasses here

Reply:
okay, makes sense